build/dependencies.props

```diff
@@ -7,11 +7,11 @@
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
     <FSharpNETSdkPackageVersion>1.0.5</FSharpNETSdkPackageVersion>
     <GoogleProtobufPackageVersion>3.1.0</GoogleProtobufPackageVersion>
-    <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15616</InternalAspNetCoreSdkPackageVersion>
+    <InternalAspNetCoreSdkPackageVersion>$(KoreBuildVersion)</InternalAspNetCoreSdkPackageVersion>
```

Comment:
KoreBuildVersion ? Why not CoreBuildVersion ? I understand this is a variable name, defined somewhere else, just wondering about why "Kore" ?

Reply:
Historical reasons. Long ago, before we branded as ASP.NET Core, our work was kalled Project K. Hence the 'K' instead of 'C'. KoreBuild, Kestrel, and [kvm](https://blogs.msdn.microsoft.com/webdev/2015/01/16/making-it-better-updates-for-asp-net-5-in-visual-studio-2015-ctp-5/). We just never kame up with a better name.